inductiva/resources/machine_groups.py

```diff
@@ -31,7 +31,7 @@ def estimate_machine_cost(machine_type: str, spot: bool = False):
     else:
         estimated_cost = instance_price.body["on_demand_price"]
 
-    return round(float(estimated_cost), 5)
+    return round(float(estimated_cost), 3)
```

Comment:
this is not the right place to round the values. They should be returned as is and only round when printing.